common/src/main/java/com/wynntils/wynn/item/UnidentifiedItemStack.java

```diff
@@ -30,6 +30,7 @@
 public class UnidentifiedItemStack extends WynnItemStack {
     private final List<Component> tooltip;
     private ItemType itemType;
+    private ItemTier itemTier;
```

Comment:
This change was not used in the end. Although, I believe this is a reasonable change, but if you want, I can revert it.

Reply:
It's ok.